### PR TITLE
Add 17.0-stable bootdisk branch

### PR DESCRIPTION
**Pull request by lzap**
SSIA I hope I got it right this time

### CODE REVIEW

File: puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/plugins/foreman_bootdisk.yaml [hunk 1]
```diff
@@ -1,5 +1,10 @@
 - project:
     name: foreman_bootdisk
     defaults: plugin
+    branch:
+      - develop:
```
Comment (mmoll) [inline puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/plugins/foreman_bootdisk.yaml:5]: Thisd should be `master`, AFAICT.

Comment (lzap) [reply puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/plugins/foreman_bootdisk.yaml:5]: *How* did you know?!? :-) Indeed.

Comment (mmoll) [reply puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/plugins/foreman_bootdisk.yaml:5]: This is the plugin's branch, `foreman_branch` is the branch name in the core repo, so `master` and `develop` need to get switched here now.

Comment (lzap) [reply puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/plugins/foreman_bootdisk.yaml]: Of course... Rebased.

**Comment by mmoll** [comment]
merged, díky @lzap!

**Pull request merged**